README.md

```diff
@@ -177,9 +177,11 @@ Distro packages already come with a `man` page. If you are building from source,
     pandoc -s -t man shellcheck.1.md -o shellcheck.1
     sudo mv shellcheck.1 /usr/share/man/man1
 
-## Travis CI
+## Travis CI & GitLab
+
+Travis CI has now intergrated ShellCheck by default, so you don't need to manually install it.
 
-Travis CI has now integrated ShellCheck by default, so you don't need to manually install it.
+GitLab does not intergrate ShellCheck by default, so you need to manually install it or pick what distro package to use.
```

Comment:
I'm not familiar with GitLab. Which distro is used? How do you pick the package?